cleanlab/classification.py

```diff
@@ -370,6 +371,11 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
           This can be useful for training differently in the final ``fit()``
           than during cross-validation.
 
+        validation_func : callable, optional
+          Optional callable function that takes two arguments, `X_val`, `y_val`.
```

Comment:
replace "." with:

, and returns a dict of keyword arguments passed into to `clf.fit()` which may be functions of the validation data in each cross-validation fold.

Reply:
done!